app/assets/stylesheets/profile.scss

```diff
@@ -113,19 +115,23 @@
   }
 
   #profile {
-    padding: 20px;
     margin-bottom: 35px;
+    border-radius: 20px 20px 0px 0px;
```

Comment:
- Shorthands of length `4` are not allowed. Value was `20px 20px 0px 0px`
- `0px` should be written without units as `0`